deps/wazuh_testing/wazuh_testing/tools/virtualization/__init__.py

```diff
@@ -0,0 +1 @@
+from .agents_dockerizer import AgentsDockerizer
```

Comment:
Yes, so Python will know that it is imported from the actual folder, this line is included in the `__init__.py` to make imports easier, it could be ignored.